7.data_harmonization/scripts/0.combine_CP_and_scDINO_features.py

```diff
@@ -16,17 +16,15 @@
 
 # define data paths for import
 cellprofiller_annotated_features_path = pathlib.Path(
-    "../../4.process_CP_features/data/annotated_data/run_20231017ChromaLive_6hr_4ch_MaxIP_sc.parquet"
+    "../../5.process_CP_features/data/3.combined_data/profiles/combined_data.parquet"
 ).resolve(strict=True)
 
 scdino_features = pathlib.Path(
-    "../../5.scDINO_analysis/1.scDINO_run/outputdir/apoptosis_timelapse/CLS_features/CLS_features_annotated.parquet"
+    "../../6.scDINO_analysis/1.scDINO_run/outputdir/apoptosis_timelapse/CLS_features/CLS_features_annotated.parquet"
 ).resolve(strict=True)
 
 # set the output path
-output_path = pathlib.Path(
-    "../data/20231017ChromaLive_6hr_4ch_MaxIP_combined_data.parquet"
-).resolve()
+output_path = pathlib.Path("../data/CP_scDINO_features/combined_data.parquet").resolve()
```

Comment:
It looks like you will have combined_data.parquet in two places; I would recommend making this file, though the folder has that relevant metadata about the file, also have that info in the file name. Totally optionally! I know in some of my repos that the SQLite files are named the same but in different folders, but that is an annoying CP issue lol. For example:

```suggestion
output_path = pathlib.Path("../data/CP_scDINO_features/combined_CP_scDINO_data.parquet").resolve()
```